README.md

```diff
@@ -41,6 +41,7 @@ Run `make test` to execute the tests
 
 ## Change log
 
+v2.3 - Add kayvee-go/validator for asserting that raw log lines are in a valid kayvee format.
```

Comment:
looks like the changelog wasn't updated for the previous v2.2. added a line for that